src/components/app-configure/AppKeyspace.vue

```diff
@@ -59,20 +59,22 @@ function handleCancel() {
 <template>
   <section name="app-branding">
     <SettingCard class="app-branding-card">
-      <template #title>Keyspace</template>
+      <template #title>Keyspace Type</template>
       <template #description>
         <div>
-          In the Arcana ecosystem, there are two options for key security: App
-          Specific (default) and Global. The latter option requires extra
-          verification during registration but allows users to use the same keys
-          across all applications. Changing options later will affect user
-          experience as wallet address will change for the user.
+          Choose from app-specific (default, highly secure) or global keyspace
```

Comment:
I recommend “between” in this case but “from” is not incorrect either.

Reply:
Done https://github.com/arcana-network/developer-dashboard/pull/216/commits/8252aad4eb1ac97179e50c6a0526221e36ce3f4f